src/madhuraMJun20/scannerClass/MissingNumberInArray.java

```diff
@@ -0,0 +1,21 @@
+/*Find the missing number in integer array of 1 to 10?*/
+package madhuraMJun20.scannerClass;
+
+public class MissingNumberInArray {
+	void findMissingNum(int[] arr, int[] num1) {
+		int sumArr = 0;
+		int totalSum = (num1.length * (num1.length + 1)) / 2;
```

Comment:
int totalSum = ((arr.length+1))) * ((arr.length+1) + 1)) / 2; You missed +1 operation.
no need to create int[] num1 = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 }; this array.

Reply:
Ok. Thanks.Done changes.